lang/src/org/partiql/lang/eval/physical/operators/ValueExpr.kt

```diff
@@ -0,0 +1,27 @@
+package org.partiql.lang.eval.physical.operators
+
+import org.partiql.lang.ast.SourceLocationMeta
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.physical.EvaluatorState
+
+/**
+ * Evaluates a PartiQL expression returning an [ExprValue].
+ *
+ * [BindingsExpr] implementations need a mechanism to evaluate such expressions, and said mechanism should
+ * avoid exposing implementation details (i.e. [org.partiql.lang.eval.physical.PhysicalPlanThunk]) of the evaluator.
+ * This implementation accomplishes that and is intended as a publicly usable API that is supported long term.
+ */
+interface ValueExpr {
```

Comment:
Any suggestions from the other reviewers for the name of this interface?  I just relased that `ValueExpr` is `ExprValue` reversed, which might be slightly confusing.

Reply:
`ValueExpr` will be renamed to `ValueExpression`

Reply:
I think you are right